packages/experimental/Shadow/package.json

```diff
@@ -25,6 +25,9 @@
     "url": "https://github.com/microsoft/fluentui-react-native.git",
     "directory": "packages/experimental/Shadow"
   },
+  "dependencies": {
+    "@fluentui-react-native/framework": "0.7.30"
```

Comment:
Needs to be updated to 0.7.31 so that we pull from the workspace and not a yarn.lock entry

Reply:
Ooh thank you that must be why the checks are failing, thought it was related to the package.json file but didn't know there was a new version